src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx

```diff
@@ -0,0 +1,561 @@
+---
+title: "NerdGraph tutorial: Data obfuscation for your logs"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+  - Logs
+metaDescription: "Create, query, and manage obfuscation rules for your log data with NerdGraph, the New Relic GraphQL explorer."
+---
+
+You control what customer data is logged with our log management service, so be sure to follow your organization's security guidelines to mask, obfuscate, or prevent sending any sensitive data. Our log management service automatically masks number patterns that appear to be for items such as credit cards or Social Security numbers. For more information, see our [security documentation](/docs/logs/get-started/new-relics-log-management-security-privacy) for log management.
+
+We also provide options for you to manage and sanitize parts of your customer data when it arrives at New Relic. This makes it easier and safer for you, rather than needing to track down sensitive data in all of your applications and try to fix them at the source.
+
+You can use NerdGraph at [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql) to create, query, and manage your data obfuscation rules for logs. NerdGraph is our GraphQL-format API explorer.
+
+**Reviewer:** What about the ability to set up obfuscation rules in the UI? I saw a front end design doc in Atlassian. Is there an ETA for that? Will both the API and the UI options be released at the same time?
+
```

Comment:
[⚠️] Please remember to remove this! Initially (MVP), we will not allow the set up of obfuscation rules in the UI, only via the API. As part of the GA, though, we plan on adding support for this in the UI as well. Not sure about the official GA date, though.

Reply:
will do. thanks for confirming!